### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: mayhem/Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline mayhem/Mayhemfile:1]: Please rename fuzz.libfuzzer

File: mayhem/fuzz.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package fuzz
```
Comment (xansec) [inline mayhem/fuzz.go]: Please rename to be more descriptive

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**